.buildkite/hooks/pre-exit

```diff
@@ -14,11 +14,13 @@ if [[ "$BUILDKITE_STEP_KEY" == *"integration-tests"* ]]; then
   SNAPSHOT=true mage integration:clean
 fi
 
-if [ -n "$GOOGLE_APPLICATION_CREDENTIALS" ]; then
-  if test -f "$GOOGLE_APPLICATION_CREDENTIALS"; then
-    rm $GOOGLE_APPLICATION_CREDENTIALS
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "buildkite-elastic-agent-integration-matrix" ]]; then
```

Comment:
we don't need this conditional here, I think the conditionals are safe here:
- if variable exists
- if file exists

Regarldess of the BK pipeline, let's keep it simple